Detectors/ITSMFT/MFT/tracking/include/MFTTracking/Definitions.h

```diff
@@ -0,0 +1,25 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+///
+/// \file Definitions.h
+/// \brief Shortcut for mathematical functions
+///
+
+#ifndef O2_MFT_DEFINITIONS_H_
+#define O2_MFT_DEFINITIONS_H_
+
+#define MATH_ABS std::abs
+#define MATH_ATAN2 std::atan2
+#define MATH_ACOS std::acos
+#define MATH_MAX std::max
+#define MATH_MIN std::min
+#define MATH_SQRT std::sqrt
```

Comment:
what is the motivation behind these macros? In my opinion, this is not really shortcutting anything.

Reply:
Yes, it's out, useless.